docusaurus/docs/Android/03-ui/04-message-components/02-message-list.mdx

```diff
@@ -135,6 +210,35 @@ messageListView.setUserReactionClickListener{ message: Message, user: User, reac
     // Listen to clicks on user reactions on the message options overlay
 }
 ```
+</TabItem>
+
+<TabItem value="java" label="Java">
+
+```java
+messageListView.setMessageClickListener((message) -> {
+    // Listen to click on message events
+});
+messageListView.setEnterThreadListener((message) -> {
+    // Listen to events when enter thread associated with a message
+});
+messageListView.setAttachmentDownloadClickListener((attachment) -> {
+    // Listen to events when download click for an attachment happens
+});
+messageListView.setUserReactionClickListener((message, user, reaction) -> {
+    // Listen to clicks on user reactions on the message options overlay
+});
+messageListView.setMessageLongClickListener((message) -> {
```

Comment:
These listeners are missing in Kotlin tab

Reply:
nice catch